server/handlers/likeHandler.ts

```diff
@@ -0,0 +1,42 @@
+import { CreateLikeRequest, CreateLikeResponse, GetLikesRequest, GetLikesResponse,  } from '../api';
+import { db } from '../datastore';
+import { ExpressHandler, Like } from '../types';
+
+export const createLikeHandler: ExpressHandler<CreateLikeRequest, CreateLikeResponse> = async(
+  req,
+  res
+) => {
+  if (!req.body.postId) {
```

Comment:
we'll also want to get the user Id from the session instead of as a parameter, let's add a TODO here?

Reply:
ok